tripmining/model/transition.py

```diff
@@ -0,0 +1,50 @@
+from geopy import distance
+
+
+class Transition:
+    """
+    Captures transitions in between the blocks
+    """
+
+    def __init__(self, from_location, to_location, time_difference):
+        self.from_location = from_location
+        self.to_location = to_location
+        self.time = self.__time(time_difference)
+        self.distance = self.__distance()
+        self.speed = self.__speed()
+
+    def __distance(self) -> float:
+        return distance.distance((self.from_location.lat, self.from_location.lng),
+                                 (self.to_location.lat, self.to_location.lng)).km
+
+    def __time(self, time_difference) -> float:
+        return time_difference.days * 24 + time_difference.seconds / 3600
+
+    def __speed(self) -> float:
+        # or some traverlers the very first checkin is not at home, so it will be at a block
```

Comment:
last_checkin_before trip and first_checkin_after_trip are removed.

Fixed in the latest PR

Reply:
This remark of mine was just related to a typo in the comment. It should be:
```
# For some travelers the very first checkin is not at home, so it will be at a block
```